roschaefer/rundfunk-mitbestimmen
# [WIP]849 send notification when broadcast is created

#### Description

 Fixes #849
#### Motivation and Context

It would be nice to send an email/slack notification informing that a new broadcast has been created.

#### How Has This Been Tested?

You will need to login to the site and create a broadcast, an easy way of doing so is as follows:
- Go to this [link](http://localhost:4200/find-broadcasts) and search for a broadcast that does not exist. 
- You'll be redirected to a page where you can create a broadcast. When you fill in the details and hit the create button you will be able to trigger the email. 
- Open mailer should send you to a link with the email details.
-  If you're not redirected, relax you can still view the emails on this [link](http://localhost:8025). This will show you all the emails that you have attempted sending. 

#### Screenshots (if appropriate):
When you have successfully created a broadcast
<img width="1440" alt="screen shot 2018-10-25 at 19 26 00" src="https://user-images.githubusercontent.com/23743836/47515582-ee029300-d88b-11e8-9bb2-87e7d3981da8.png">
Just before you save the broadcast
<img width="1440" alt="screen shot 2018-10-25 at 19 25 42" src="https://user-images.githubusercontent.com/23743836/47515608-feb30900-d88b-11e8-91c8-91c0162440d0.png">

#### Checklist:
